EventListener/TemplateListener.php

```diff
@@ -59,7 +59,10 @@ public function onKernelController(FilterControllerEvent $event)
             return;
         }
 
-        $template->setOwner($controller = $event->getController());
+        $controller = $event->getController();
+        if (is_array($controller)) {
+            $template->setOwner($controller);
```

Comment:
Please be ware that this would break `$owner = $template->getOwner();` in `onKernelView`. It should probably do a sanity check now and skip the view listener as the controller has changed.